ui/js/models/perf/alerts.js

```diff
@@ -25,16 +25,9 @@ treeherder.factory('PhAlerts', [
 
             // for talos only, automatically add related branches
             if (performanceFrameworkId === 1) {
-                _.forEach(thPerformanceBranches, function(performanceBranch) {
-                    if (performanceBranch !== alertRepository) {
-                        url += "&series=[" + [performanceBranch, signature, 0] + "]";
-                    }
-                });
-                if (alertRepository === "mozilla-beta") {
-                    url += "&series=[" + ["mozilla-aurora", signature, 0] + "]";
-                }
+                const branches = (alertRepository === "mozilla-beta") ? ['mozilla-inbound'] : thPerformanceBranches.filter(branch => branch !== alertRepository);
+                url += branches.map(branch => `&series=[${branch},${signature},0]`).join("");
             }
-
```

Comment:
not that big a deal but I wouldn't remove this newline (maybe you can revert that before merging?)